chapters/5-processing.tex

```diff
@@ -325,15 +328,15 @@ \section{Data quality assurance}
 \subsection{Implementing high-frequency quality checks}
 
 % What are HFCs
-A key advantage of continuous electronic data ingestion methods,
+A key advantage of continuous electronic data intake methods,
```

Comment:
ingestion is the correct term

Reply:
I hate it though, it makes me so squeamish 😂

```suggestion
A key advantage of continuous electronic data ingestion methods,
```